exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,7 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+let totalPeople = numberOfStudents+numberOfMentors;
+console.log(
+`Percentage students: ${Math.round(100*numberOfStudents/totalPeople)}%
+Percentage mentors: ${Math.round(100*numberOfMentors/totalPeople)}%`
+);
```

Comment:
Again all three variables here should be stored as const, avoid var if you can, it's old syntax

Reply:
Done 👍 Thank You for coution sir